### PR TITLE
Web single file loading and embedded audio

### PR DESCRIPTION
**Pull request by patrick-volograms**
## Purpose of this PR

* Add single-file-loading functionality to the web player
* Add access to embedded audio data to web
* Update documentation

***

## Changes made in this PR

* Added single-file-loading function to `wasm_vol_geom.c` (and convenience interface to `pre.js`) 
* Added embedded audio access functions to `wasm_vol_geom.c` (and convenience interfaces to `pre.js`) 
* Added `get_audio_data()` function to `pre.js` which makes is even easier to access audio data
* Added info of new functions to web player readme

***

### Build Command

```sh
cd wasm
./build-mjs.sh && ./build-js.sh
```

### Checklist

- [x] Desktop player

https://user-images.githubusercontent.com/68862565/235972839-87ac69b1-54fd-4515-9ba6-1b8a0166889f.mov

##### Build type

- [ ] Module built with `build-mjs.sh`
- [ ] Code built with `build-js.sh`

***

## Risk Introduced

- [ ] Not much consideration was added to backward compatibility of vol format versions

***

# Pre-Merge Checklist

- [ ] No conflicts with destination branch
- [ ] Reviewed your own PR and checked your changed code
- [ ] Invited reviewers (ideally +2)
- [ ] All questions/comments answered (where necessary)
- [ ] Approvals received after latest changes (ideally 2+)

***

### CODE REVIEW

File: wasm/README.md [hunk 1]
```diff
@@ -151,11 +159,18 @@ Parameters:
 
 #### `Module.create_file_info(hdr: String, seq: String): Boolean`
 
-Open and initialse a vologram. Returns `true` if successful and `false` otherwise.
+Open and initialses a vologram from separate header and sequence files. Returns `true` if successful and `false` otherwise.
```
Comment (capnramses) [inline wasm/README.md:162]: Opens

File: wasm/README.md [hunk 2]
```diff
@@ -151,11 +159,18 @@ Parameters:
...
 
 Parameters:
 * `hdr: String` - Path of vologram header file
 * `seq: String` - Path of vologram sequence file
+* 
+#### `Module.create_single_file_info(file: String): Boolean`
+
+Open and initialses a vologram from a single file. Returns `true` if successful and `false` otherwise.
```
Comment (capnramses) [inline wasm/README.md:170]: Opens

File: wasm/pre.js [hunk 3]
```diff
@@ -230,7 +241,13 @@ Module.initVologramFunctions = (containerObject) => {
 		return [glFmt, basisFmt];
 	};
 
+	insertObject["get_audio_data"] = () => {
```
Comment (capnramses) [inline wasm/pre.js:244]: nice

**Comment by capnramses** [comment]
good! i think we just need some examples/ with basis/audio

**Review by capnramses — APPROVED** [review]

**Review by ondrj006 — APPROVED** [review]
All looking good!

**Pull request merged**